app/uk/gov/hmrc/gform/models/ExpandUtils.scala

```diff
@@ -50,7 +50,7 @@ object ExpandUtils {
 
   def expandSmartString(smartString: SmartString, index: Int, ids: List[FormComponentId]): SmartString = {
     val interpolations: List[Expr] = smartString.interpolations.map(expr => ExprUpdater(expr, index, ids))
-    smartString.copy(interpolations = interpolations).replace("$n", index.toString)
+    smartString.copy(interpolations = interpolations).replace("$n", "")
```

Comment:
This would break `$n` functionality, we cannot do that.

Unfortunately I haven't any suggestion how to fix this issue.